.github/workflows/azwi-e2e.yaml

```diff
@@ -55,16 +49,15 @@ jobs:
           echo "SERVICE_ACCOUNT_NAME=${SERVICE_ACCOUNT_NAMESPACE}-sa" >> "${GITHUB_ENV}"
       - name: Create kind cluster
         run: |
-          # create a kind cluster with predefined signing keys
-          echo ${{ secrets.SERVICE_ACCOUNT_SIGNING_KEY }} | base64 -d > sa.key
-          echo ${{ secrets.SERVICE_ACCOUNT_KEY }} | base64 -d > sa.pub
+          openssl genrsa -out sa.key 2048
+          openssl rsa -in sa.key -pubout -out sa.pub
```

Comment:
If we generate the signing keys in CI, we'll need to upload the discovery document and jwks to storage account? I'm assuming this was just a debug step and this is not what we intend to do?

Reply:
Since we are not doing any token exchange test, the content of the discovery document shouldn't matter. This way, we can eliminate `SERVICE_ACCOUNT_SIGNING_KEY` and `SERVICE_ACCOUNT_KEY` variables.

Reply:
Makes sense!